src/main/java/com/zaxxer/hikari/HikariConfig.java

```diff
@@ -848,7 +867,7 @@ private void logConfiguration()
                value = dsProps;
             }
             value = (prop.contains("password") ? "<masked>" : value);
-            LOGGER.debug((prop + "................................................").substring(0, 32) + (value != null ? value : ""));
+            LOGGER.debug((prop + "................................................").substring(0, 32) + value);
```

Comment:
That's fine.  Please integrate the change above to quote strings in the debug output.  Users of the pool do not know what is occurring internally (trimming, no trimming, some trimming) but the debug output can make it clear.